taxi/forms.py

```diff
@@ -0,0 +1,38 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+
+class LicenseValidationMixin:
+    def clean_license_number(self) -> str:
+        license_number = self.cleaned_data.get("license_number")
+        if not (
+                license_number[:3].isalpha()
+                and license_number[:3].isupper()
+                and license_number[3:].isdigit()
+                and len(license_number) == 8
+        ):
+            raise forms.ValidationError(
+                "License number must consist of "
+                "3 uppercase letters followed by 5 digits."
+            )
+        return license_number
+

```

Comment:
let's refactor this if and use:
if (a and b and c and d): return number
else raise error

In general if you have a few if statements that should check data for correctness before return statement, then it's a correct approach but as you merged all conditions, then it's better to do as I've explained above it order to increase code readability